setup.cfg

```diff
@@ -51,6 +51,7 @@ exclude = .git,__pycache__,venv,.venv
 [flake8]
 max-line-length = 120
 exclude = .git,__pycache__,venv,.venv
+per-file-ignores = __init__.py:F401
```

Comment:
Added this in as well, not worth it's own PR. This ignores a specific warning kind that shouldn't trigger: unused imports in __init__.py files.